webpack/scenes/ContentViews/Details/Filters/ContentViewFilterDetails.js

```diff
@@ -44,6 +45,7 @@ const ContentViewFilterDetails = () => {
         {
           {
             package_group: <CVPackageGroupFilterContent cvId={cvId} filterId={filterId} />,
+            rpm: <CVRpmFilterContent filterId={filterId} inclusion={inclusion} />,
           }[type] || <Loading />
```

Comment:
This is a very odd pattern to me, and feels a bit too imperative for the declarative nature of JSX.  Would you consider rewriting to move this logic outside the JSX?

Reply:
This is new to me too..Trying out a few other options..